File: Arrays/clonerTableau.js
/**
 * Permet de cloner un tableau dans un tableau.
 * @param {Array} arr - Le tableau à cloner.
 * @returns {Array} Le tableau avec le clone.
 */
const clonerTableau = (arr) => [...arr, arr];

console.log(clonerTableau([0, 0, 0])); // [ 0, 0, 0, [ 0, 0, 0 ] ]

File: Python/Arrays/clonerTableau.py
def cloner_tableau(arr: list) -> list:
    """
    Description:
        Permet de cloner un tableau dans un tableau.
        L'opérateur ( * ) permet de déballer un itérable pour en créer un nouveau.

    Paramètres:
        arr: {list} -- Tableau à cloner.
    
    Retourne:
        {list} -- le tableau cloné.

    Exemple:
        >>> arr = [0, 0, 0]]
        >>> cloner_tableau(arr)
        [0, 0, 0, [0, 0, 0]]
    """
    return [*arr, arr]